components/brave_ads/core/internal/account/utility/redeem_confirmation/reward/redeem_reward_confirmation_unittest.cc

```diff
@@ -31,6 +31,12 @@ namespace brave_ads {
 
 class BraveAdsRedeemRewardConfirmationTest : public UnitTestBase {
```

Comment:
Can we please add a test to check that the delay matches `kFetchPaymentTokenAfter` so that we know a dev has not changed `kFetchPaymentTokenAfter` to something else.

See `TimeDelta NextMainThreadPendingTaskDelay() const;` in `task_environment_`; Would be great to add a helper for this in `unittest_base`.

Reply:
Fixed